admin_scripts/ubuntu/change_computer_name.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# This script will change the name of the computer, it will not
+# change how it looks in the admin. There will be no validation that this parameter is a valid name.
```

Comment:
This isn't strictly relevant to _this_ PR, but it's on my mind: we should update the client modules so that the system's `hostname` is sent whenever it checks in, so that we _can_ keep the system name and the admin name in sync.

Reply:
Yes I agree. I have created an issue #32578